db/institutions.yml

```diff
@@ -680,6 +680,39 @@ enthought:
   updated: 2020-12-29 04:16:46.030159
   uuid: 358f4e2b-e0dc-44e6-8cf4-605409118494
   zip: '78701'
+epfl:
+  aka:
+    - EPFL,
+    - EPFL
+    - Valais,
+    - EPFL
+    - Wallis
+  city: Sion
+  country: Switzerland
+  date: 2023-09-14
+  departments:
+    isic:
+      name: Institute of Chemical Sciences and Engineering
+      aka:
+        - Institut
+        - des
+        - sciences
+        - et
+        - ingénierie
+        - chimiques
+  name: École Polytechnique Fédérale de Lausanne
+  schools:
+    sb:
+      name: school of basic science
+      aka:
+        - faculté
```

Comment:
same here

Reply:
ah I missed that. I corrected the file and committed and pushed it.